Translation/en/Text/Main/CharacterMaker.highres.txt

```diff
@@ -0,0 +1,49 @@
+//*******************************************************************
+//*******************************************************************
+// Note: This file contains only UI elements for Chara Maker
+//       that need special handling based on resolution
+//*******************************************************************
+//*******************************************************************
+
+//Scope of the file: Only applies to the Chara Maker for Koikatsu Sunshine main game in resolutions >= 1080p
+
+#set required-resolution width > 1919
```

Comment:
I don't like the idea to implement this now, it's clearly a bug in XUA. I prefer to wait for a proper fix before using the new feature